app/facades/tagging_spreadsheet_facade.rb

```diff
@@ -0,0 +1,21 @@
+class TaggingSpreadsheetFacade < SimpleDelegator
```

Comment:
Out of interest - why have we opted to call this a 'facade' when all of these methods are related to view-layer presentation? Seems like `presenter` would be a more fitting, less obscure naming choice.